lang/en.yml

```diff
@@ -1,78 +1,89 @@
 en:
-  AssetTableField:
+  SilverStripe\AssetAdmin\Controller\AssetAdmin:
```

Comment:
I know the upgrader doesn't support this, but is it possible to sort these keys by the FQN? Just for the master en.yml is fine. :)